### DISCUSSION TITLE
Can I upload reports using only jmeter ?

### DISCUSSION THREAD

**Discussion by OguzhanC07** [ludeknovy/jtl-reporter#80] [Q&A] [answered]
I don't use taurus or locust. I only use pure Jmeter app. When I try to upload csv or jtl report it wants kpi jtl file but I don't have that. My question is can I upload my report using only jmeter app ?

**Comment by ludeknovy** [comment, marked as answer]
Hi @OguzhanC07, yes, absolutely - please check: https://jtlreporter.site/docs/integrations/jmeter

Taurus just makes it a bit easier to automate, but its not required.

**Reply by ludeknovy** [reply]
@OguzhanC07 please let me know if it worked for you?

**Reply by OguzhanC07** [reply]
Sorry I forgot to reply you. We started to use. It works like a charm thanks

**Reply by ludeknovy** [reply]
@OguzhanC07 Great! Glad you solved! Please could you mark my answer as "answered"?
